src/NuGet.Jobs.Db2AzureSearch/Job.cs

```diff
@@ -1,26 +1,80 @@
+using NuGet.Services.AzureSearch.Wrappers;
 
 namespace NuGet.Jobs
 {
     public class Job : JsonConfigurationJob
     {
-        public override Task Run()
+        private const string Db2AzureSearchSectionName = "Db2AzureSearch";
+        private const string SearchIndexKey = "SearchIndex";
+        private const string HijackIndexKey = "HijackIndex";
+
+        public override async Task Run()
         {
-            return Task.CompletedTask;
+            var db2AzureSearch = _serviceProvider.GetRequiredService<IDb2AzureSearchCommand>();
+            await db2AzureSearch.ExecuteAsync();
         }
 
         protected override void ConfigureAutofacServices(ContainerBuilder containerBuilder)
         {
+            containerBuilder
+                .Register(c =>
+                {
+                    var serviceClient = c.Resolve<ISearchServiceClientWrapper>();
+                    var options = c.Resolve<IOptionsSnapshot<Db2AzureSearchConfiguration>>();
+                    return serviceClient.Indexes.GetClient(options.Value.SearchIndexName);
+                })
+                .SingleInstance()
+                .Keyed<ISearchIndexClientWrapper>(SearchIndexKey);
+
+            containerBuilder
+                .Register(c =>
+                {
+                    var serviceClient = c.Resolve<ISearchServiceClientWrapper>();
+                    var options = c.Resolve<IOptionsSnapshot<Db2AzureSearchConfiguration>>();
+                    return serviceClient.Indexes.GetClient(options.Value.HijackIndexName);
+                })
+                .SingleInstance()
+                .Keyed<ISearchIndexClientWrapper>(HijackIndexKey);
+
+            containerBuilder
+                .Register<IDb2AzureSearchCommand>(c => new Db2AzureSearchCommand(
+                    c.Resolve<INewPackageRegistrationProducer>(),
+                    c.Resolve<IIndexActionBuilder>(),
+                    c.Resolve<ISearchServiceClientWrapper>(),
+                    c.ResolveKeyed<ISearchIndexClientWrapper>(SearchIndexKey),
+                    c.ResolveKeyed<ISearchIndexClientWrapper>(HijackIndexKey),
+                    c.Resolve<IOptionsSnapshot<Db2AzureSearchConfiguration>>(),
+                    c.Resolve<ILogger<Db2AzureSearchCommand>>()));
         }
 
         protected override void ConfigureJobServices(IServiceCollection services, IConfigurationRoot configurationRoot)
         {
+            services.Configure<Db2AzureSearchConfiguration>(configurationRoot.GetSection(Db2AzureSearchSectionName));
+
+            services.AddSingleton<ISearchServiceClientWrapper>(p =>
+            {
+                var options = p.GetRequiredService<IOptionsSnapshot<Db2AzureSearchConfiguration>>();
+                var inner = new SearchServiceClient(
+                    options.Value.SearchServiceName,
+                    new SearchCredentials(options.Value.SearchServiceApiKey));
+                return new SearchServiceClientWrapper(inner);
```

Comment:
>SearchServiceClientWrapper [](start = 27, length = 26)

Super super nit: it seems like you could add `SearchServiceClient` and `SearchServiceClientWrapper`. Feel free to ignore :)